.github/workflows/tests.yml

```diff
@@ -193,7 +193,7 @@ jobs:
       fail-fast: ${{fromJson(needs.setup_tests.outputs.fail-fast)}}
     runs-on: ${{matrix.runner}}
     container: ${{matrix.container}}
-    timeout-minutes: ${{ (matrix.runner == 'ubuntu-latest' && 360) || fromJson(needs.setup_tests.outputs.timeout-minutes) }}
+    timeout-minutes: ${{ (runner.os == 'Linux' && 4320) || fromJson(needs.setup_tests.outputs.timeout-minutes) }}
```

Comment:
I don't think this check works outside of `steps`.

Reply:
Ah right. Good catch